playground/src/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+window.React = React;
```

Comment:
Not certain why this is needed yet. Maybe a parcel thing.

Reply:
Wasn't it an issue only because Babel config couldn't be loaded?

Reply:
I'll have to check again.

Reply:
So to solve this, I had to explicitely add the pragma config to the `.babelrc`. I'm not sure why this is needed though as in theory it's already done by the wordpress default preset. 🤔